ouroboros-consensus-diffusion/ouroboros-consensus-diffusion.cabal

```
@@ -318,10 +316,68 @@ test-suite consensus-test
     unstable-diffusion-testlib,
     vector,
 
+test-suite consensus-test
+  import: common-test
+  type: exitcode-stdio-1.0
+  hs-source-dirs: test/consensus-test
+  main-is: Main.hs
+
+  build-depends:
+    QuickCheck,
+    base,
+    binary,
+    bytestring,
+    cardano-crypto-class ^>=2.2,
+    cardano-ledger-core,
+    cardano-slotting:{cardano-slotting, testlib},
+    cardano-strict-containers,
+    containers,
+    contra-tracer,
+    directory,
+    fs-api ^>=0.4,
+    fs-sim ^>=0.4,
+    hashable,
+    io-classes:{io-classes, si-timers, strict-stm},
+    io-sim,
+    mempack,
+    mtl,
+    nothunks,
+    ouroboros-consensus,
```

```suggestion
    ouroboros-consensus:{ouroboros-consensus, unstable-consensus-testlib},
```

It seems it also needs to import this other package.